crates/jstzd/src/task/jstzd.rs

```diff
@@ -0,0 +1,115 @@
+use super::{octez_node::OctezNode, Task};
+use anyhow::Result;
+use async_dropper_simple::AsyncDrop;
+use async_trait::async_trait;
+use octez::r#async::node_config::OctezNodeConfig;
+use tokio::task::JoinHandle;
+
+struct Jstzd {
+    octez_node: OctezNode,
+}
+
+#[derive(Clone)]
+pub struct JstzdConfig {
+    octez_node_config: OctezNodeConfig,
+}
+
+impl JstzdConfig {
+    pub fn new(octez_node_config: OctezNodeConfig) -> Self {
+        Self { octez_node_config }
+    }
+}
+
+#[async_trait]
+impl Task for Jstzd {
+    type Config = JstzdConfig;
+
+    async fn spawn(config: Self::Config) -> Result<Self> {
+        Ok(Self {
+            octez_node: OctezNode::spawn(config.octez_node_config.clone()).await?,
+        })
+    }
+
+    async fn kill(&mut self) -> Result<()> {
+        self.octez_node.kill().await
```

Comment:
Note: When the other octez processes are handled, we will want to kill them concurrently.